jwst/lastframe/lastframe_step.py

```diff
@@ -7,7 +7,8 @@ class LastFrameStep(Step):
     """
     LastFrameStep: This is a MIRI specific task to correct the last frame by
     subtracting the array contained in the last frame reference file from the
-    last frame.
+    last frame.  If the number of groups is greater than 1, the GROUP data
+    quality flags for the final group will be set to DO_NOT_USE.
     """
 
     reference_file_types = ['lastframe']
```

Comment:
In addition to flagging the last group, the changes also need to do away with all other existing corrections applied to the data, so that only the last group flagging occurs. This means modifying lastframe_step.py to remove all references to the 'lastframe' reference file, because it won't be used any more.